packages/react-utilities/src/hooks/useEffectAfterMount.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef, DependencyList, EffectCallback } from 'react';
+
+/**
+ * Similar to React.useEffect, only difference is that the callback is not invoked on mount.
+ *
+ * A common use case is the invocation of external callbacks to ensure controlled component.
+ * @example
+ *  useEffectAfterMount(() => {
+ *    props.onChange?.(value)
+ *  }, [value, props.onChange])
+ * @param callback effect to be invoked when dependencies change
+ * @param dependencies dependency list to determine when the effect should run
+ */
+export function useEffectAfterMount(callback: EffectCallback, dependencies: DependencyList = []) {
+  const justMounted = useRef(true);
+  useEffect(() => {
+    if (!justMounted.current) {
+      return callback();
```

Comment:
does the callback also need to go in the dependencies ?

Reply:
No. if it's added at the depdnency it would be like if the useEffect callback should be added in the dependency.